.cbmc-batch/jobs/aws_byte_buf_from_c_str/Makefile

```diff
@@ -1,8 +1,36 @@
-include ../Makefile
-
-proofs.goto:
-	cd ../../..; \
-	goto-cc .cbmc-batch/byte_buf_proofs.c source/byte_buf.c \
-	source/array_list.c source/common.c source/error.c -Iinclude/ \
-	-I.cbmc-batch/ -o .cbmc-batch/jobs/aws_byte_buf_from_c_str/proofs.goto \
-	--function aws_byte_buf_from_c_str_verify
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+#########
+# if Makefile.local exists, use it. This provides a way to override the defaults
+sinclude ../Makefile.local
+#otherwise, use the default values
+include ../Makefile.local_default
+
+#########
+ENTRY = aws_byte_buf_from_c_str
+HARNESS_NAME = $(ENTRY)_harness.c
```

Comment:
Would it be simpler to just make the filename ```harness.c``` by default? The directory already specifies the entry name?